### PR TITLE
Adds more gas to the N2 and Airmix tanks.

### PR DESCRIPTION
**Pull request by francinum**
:cl: Francinum
tweak: Adds more gas to the Airmix and N2 tanks.
/:cl:
Maybe now they won't drain after being exposed to a moderately sized breach.

These are next to the Airmix tank.

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
Can you provide a screenshot?

File: _maps/map_files/Trailblazer/Trailblazer.dmm [hunk 1]
```diff
@@ -8439,6 +8426,20 @@
...
+	id_tag = "air_sensor"
+	},
+/obj/structure/window/reinforced,
+/obj/structure/window/reinforced{
+	dir = 1
+	},
+/obj/machinery/portable_atmospherics/canister/air,
+/turf/open/floor/engine/air{
+	initial_gas_mix = "o2=120000;n2=480000;TEMP=293.15"
+	},
```
Comment (SgtxRootbeer) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: Looks good enough for me... Can't officially declare it kosher until I get home and open it in fastdmm, but, looks good to me!

Comment (francinum) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: the values used by default are... Strange.

Comment (francinum) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: And don't add up to a complete 80-20 split of the standard value.

**Comment by SgtxRootbeer** [comment]
made redundant by #172

**Pull request closed without merging**